mne/channels/channels.py

```diff
@@ -656,9 +656,9 @@ class UpdateChannelsMixin(object):
     def pick_types(self, meg=False, eeg=False, stim=False, eog=False,
                    ecg=False, emg=False, ref_meg='auto', misc=False,
                    resp=False, chpi=False, exci=False, ias=False, syst=False,
-                   seeg=False, dipole=False, gof=False, bio=False, ecog=False,
-                   fnirs=False, csd=False, include=(), exclude='bads',
-                   selection=None, verbose=None):
+                   seeg=False, dbs=False, dipole=False, gof=False, bio=False,
```

Comment:
dbs just be added just before include to avoid risk of crash if someone passed parameters without names